### PR TITLE
ADD transaction and Group By queries - Update / Delete operations

### PR DESCRIPTION
**Pull request by Ewoskyx**
### Transactions 
In this feature I did;

-Add Transactions (Begin - Rollback)
-Add UPDATE and DELETE while transactions
-Add new animals 
-Add a new column (species)
-Add new queries according to the project requirements

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -7,8 +7,73 @@ INSERT INTO
...
+SELECT name, species 
+  FROM animals;
+
+/* Delete all records in the animals table and Rollback */
+BEGIN WORK;
+
+DELETE FROM animals;
+
+ROLLBACK WORK;
+
```
Comment (mmsesay) [inline data.sql:65]: - [x] I believe you forgot to verify the changes. You can try to add it.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Ewoskyx, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Ewoskyx) [reply data.sql:65]: Ups, I didn't notice the verification after rollback. Thank you for pointing this out.

**Review by rustam-tolipov — APPROVED** [review]
## Hi @Ewoskyx 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**